dali/python/nvidia/dali/plugin/tf.py

```diff
@@ -230,6 +251,29 @@ def _insert_experimental_member(member, name):
     setattr(experimental_module, name, member)
 
 
+def _get_layout_from_pipeline(input_name, name_es_map):
+    layout = name_es_map[input_name]._layout
+    return layout if layout is not None else ""
```

Comment:
```suggestion
    return layout or ""
```
would do, I think

Reply:
I think you're right. Done.

Reply:
But actually, it's leftover function, removed.